taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -12,16 +11,13 @@ const ListTranslations = ({ nodeObject, languageNames, setNodeObject }) => {
             }
     })
 
-    // Add main language before setting it as state variable
-    let duplicateToBeRendered = {...toBeRendered};
-    duplicateToBeRendered[nodeObject.main_language] = nodeObject["tags_"+nodeObject['main_language']];
-    const [changedObj, setChangedObj] = useState(duplicateToBeRendered);
-
     // Helper function used for changing state
     function changeData(key, index, obj) {
```

Comment:
is the obj really an `{}` ? from what happens below I'd think it is a string?

Reply:
Yeah, it's a string. Sorry for the bad naming of the variable.